test/regressions/fixtures/TextField/FilledHiddenLabelInputAdornment.js

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import Box from '@material-ui/core/Box';
+import TextField from '@material-ui/core/TextField';
+import InputAdornment from '@material-ui/core/InputAdornment';
+
+export default function FilledHiddenLabelInputAdornment() {
+  return (
+    <Box
+      component="form"
+      sx={{
+        '& .MuiTextField-root': { m: 1, width: '25ch' },
+      }}
+      noValidate
+      autoComplete="off"
+    >
+      <Box sx={{ display: 'flex', alignItems: 'center' }}>
+        <label htmlFor="filled-hidden-label-small">Hidden label</label>
```

Comment:
If we can remove these lines and still reproduce the visual effect, I would encourage we do. The less code we have, the better (less inertial with breaking changes).

Reply:
Simplified